### ISSUE TITLE
PyQt6 build was canceld

### ISSUE THREAD

**Issue by JakobDev** [flathub/com.riverbankcomputing.PyQt.BaseApp#1]
The [PyQt6 Build](https://flathub.org/builds/#/builders/21/builds/6745) was canceld. 

These are the last Lines
```
]2;flatpak-builder: Cleanup pyqtError: module pyqt: Child process exited with code 2
FB: Unmounting read-only fs: fusermount -uz /srv/buildbot/worker/build-aarch64-5/build/.flatpak-builder/rofiles/rofiles-obdDSV
dbus-daemon[3031530]: Reloaded configuration
```

After that, it was hanging until it was canceld. I'm not sure, if the cleanup error causes this or if it was just some random freeze, but the cleanup error should be fixed.

**Comment by JakobDev** [comment]
I have made a empty commit on the 6.2 branch to trigger a rebuild. This will show, if it is just some random freeze or if it can be reproduced.

**Comment by hfiguiere** [comment]
tip: you can retrigger builds from https://flathub.org/builds/ 

all you need is to login with Github and if your account have commit access it can do it

**Comment by JakobDev** [comment]
Thanks. I didn't know that.

But it looks like aarch64 is hanging again while x86_64 has no problems., but I don't know what causes this. You need to take a look at it.

**Comment by hfiguiere** [comment]
I can't. 

@barthalion can.

But maybe just patience is needed.

**Comment by tinywrkb** [comment]
See #2.  
Also, see my comment [here](https://github.com/flathub/com.riverbankcomputing.PyQt.BaseApp/commit/7cd271ca38fbe7d752eb2a4910339ed0d38110db).

**Comment by tinywrkb** [comment]
Closed by #2.
